### PR TITLE
WDE002906/MEG5001-0300 update

### PR DESCRIPTION
**Pull request by Nailik**
WDE002906/MEG5001-0300
- updated to modern extend
- added support for `switchActions`
- did NOT add support for `indicatorMode` because it's not supported due to this being a battery device

### CODE REVIEW

**Comment by Nailik** [comment]
In theory it works, but is it by design that an action is follow by an empty one? @Koenkk 
Action triggered by user: `"action":"off_left"`
Followed by `"action":""`

```
Info 2024-04-02 11:59:19MQTT publish: topic 'zigbee2mqtt/wohnzimmer_remote', payload '{"action":"off_left","battery":100,"linkquality":204,"switch_actions_left":"light","switch_actions_left_btn":"dimmer_opposite","switch_actions_right":"dimmer","switch_actions_right_btn":"dimmer_opposite"}'
Info 2024-04-02 11:59:19MQTT publish: topic 'zigbee2mqtt/wohnzimmer_remote', payload '{"action":"","battery":100,"linkquality":204,"switch_actions_left":"light","switch_actions_left_btn":"dimmer_opposite","switch_actions_right":"dimmer","switch_actions_right_btn":"dimmer_opposite"}'
```

**Comment by Koenkk** [comment]
Yes, see https://www.zigbee2mqtt.io/guide/faq/#why-is-the-action-property-always-empty. Can I merge this?

**Comment by Nailik** [comment]
yes can be merged i'll update the docs

File: src/devices/schneider_electric.ts [hunk 1]
```diff
@@ -1028,26 +1028,17 @@ const definitions: Definition[] = [
...
-        configure: async (device, coordinatorEndpoint, logger) => {
-            // When in 2-gang operation mode, unit operates out of endpoints 21 and 22, otherwise just 21
-            const leftButtonsEndpoint = device.getEndpoint(21);
-            await reporting.bind(leftButtonsEndpoint, coordinatorEndpoint, ['genOnOff', 'genLevelCtrl', 'genPowerCfg']);
-            const rightButtonsEndpoint = device.getEndpoint(22);
-            await reporting.bind(rightButtonsEndpoint, coordinatorEndpoint, ['genOnOff', 'genLevelCtrl']);
-            await reporting.batteryPercentageRemaining(leftButtonsEndpoint);
-        },
+        extend: [
+            deviceEndpoints({endpoints: {'right': 21, 'left': 22}}),
```
Comment (Koenkk) [inline src/devices/schneider_electric.ts:1037]: Also add `battery` here

File: src/devices/schneider_electric.ts [hunk 2]
```diff
@@ -1028,26 +1028,17 @@ const definitions: Definition[] = [
     },
     {
         zigbeeModel: ['FLS/SYSTEM-M/4'],
-        model: 'WDE002906',
+        model: 'WDE002906/MEG5001-0300',
         vendor: 'Schneider Electric',
         description: 'Wiser wireless switch 1-gang or 2-gang',
         fromZigbee: [fz.command_on, fz.command_off, fz.command_move, fz.command_stop, fz.battery],
```
Comment (Koenkk) [inline src/devices/schneider_electric.ts:1034]: I expect you can completely drop `fromZigbee`

Comment (Nailik) [reply src/devices/schneider_electric.ts]: yes i removed from and to Zigbee

File: src/devices/schneider_electric.ts (hunk 1 above)
Comment (Nailik) [reply src/devices/schneider_electric.ts:1036]: it was already there i rearrange it for better readability

**Pull request merged**